draft-ietf-quic-recovery.md

```diff
@@ -521,6 +521,10 @@ is available, or if the network changes, the initial RTT SHOULD be set to 500ms,
 resulting in a 1 second initial handshake timeout as recommended in
 {{?RFC6298}}.
 
+A connection MAY use the delay between sending a PATH_CHALLENGE and receiving
+a PATH_RESPONSE to seed initial_rtt for a new path, but MUST not be considered
```

Comment:
Consider adding "the delay" -- "..., but _the delay_ MUST not be considered..."

In the current form, the sentence suggests that the connection must not be considered an RTT sample.

Reply:
Thanks, done.